NebulaWorld/SimulatedWorld.cs

```diff
@@ -371,21 +370,26 @@ public static void OnDronesDraw()
         public static void OnDronesGameTick(long time, float dt)
         {
             double tmp = 1e10; //fake energy of remote player, needed to do the Update()
-            double tmp2 = 1;
-            //Update drones positions based on their targets
+            double tmp2 = 1;
+            //Update drones positions based on their targets
+            var prebuildPool = GameMain.localPlanet.factory.prebuildPool;
+
             foreach (KeyValuePair<ushort, RemotePlayerModel> remoteModel in remotePlayersModels)
-            {
-                MechaDrone[] drones = remoteModel.Value.PlayerInstance.mecha.drones;
-                int droneCount = GameMain.mainPlayer.mecha.droneCount;
+            {
+                Mecha remoteMecha = remoteModel.Value.MechaInstance;
+                MechaDrone[] drones = remoteMecha.drones;
+                int droneCount = remoteMecha.droneCount;

```

Comment:
This was changed from mainPlayer.droneCount, just in case players could have differing drone counts.

Reply:
@Baldie-dev Was the `remoteMecha.droneCount` variable synced ?

Reply:
YEs, it is, I checked: https://github.com/Therzok/nebula/blob/1bfe56fd223f327ec4cdecf381f6af87d5a6f2c2/NebulaWorld/SimulatedWorld.cs#L138